### PR TITLE
Feat: data_model - creation

### PR DESCRIPTION
**Pull request by oxedions**
Data model, to be a reference.

### CODE REVIEW

File: resources/data_model.md [hunk 1]
```diff
@@ -0,0 +1,288 @@
...
+j2_current_iceberg: Iceberg host is member of. (only one) (icebergX)
+j2_current_iceberg_number: Iceberg number host is member of. (X)
+j2_current_iceberg_network: Iceberg network host is member of (iceX)
+
+j2_node_main_resolution_network: Main resolution network. The network on which host can be ping by direct name. (ex: ping c001).
+j2_node_main_network: Main network. The network used by Ansible to deploy configuration (related to ssh).
+j2_node_main_network_interface: Main network interface. Same as main network, but provides interface name instead of network.
+j2_management_networks: List of management networks.
+
+j2_role_variables: Used in core roles. Sort role vars to grab current OS related ones.
```
Comment (btravouillon) [inline resources/data_model.md:287]: to remove?

File: resources/data_model.md [hunk 2]
```diff
@@ -0,0 +1,288 @@
...
+
+```yaml
+iceberg_naming:
+equipment_naming:
+management_networks_naming:
+master_groups_naming:
+managements_group_name:
+```
+
+## Advanced usage
```
Comment (btravouillon) [inline resources/data_model.md:268]: You should specify those are internal variables for stack developers only.

File: resources/data_model.md (hunk 1 above)
Comment (oxedions) [reply resources/data_model.md:287]: Of course !! But I don't get it, why do I do not see it on my local clone ? I even cloned again, and this part of the file is not present in the oxedions/data_model branch. I will do it on github then.

**Comment by oxedions** [comment]
Should be ok now.

**Review by btravouillon — APPROVED** [review]
Looks good to me. 👍

**Pull request merged**